### ISSUE TITLE
FEATURE: Support basic navigation for CLI menus with PS3 controller

### ISSUE THREAD

**Issue by free5ty1e** [free5ty1e/primestationone#340]
...perhaps via joy2key or the new inputstation?

**Comment by free5ty1e** [comment]
This would enable users to be able to select netplay, resolution, and emulator via the "x" menu that is available before most emulators launch from emulationstation.  This occurs after Emulationstation has released the controller interface and has turned it over to the terminal.  

We don't want to require any keyboard at all unless the user wants to play a game that actually requires one (such as old text adventures and the like).  

The point of the Primestation One is to be able to massively retrogame from the comfort of your couch, without getting up to change cartridges or wiring or inputs, with optional netplay capabilities.

Hmm, that last line sounds good.  I think I'll add it to the Readme.

**Comment by pr1ntf** [comment]
Could this be as simple as temporarily mapping the arrow keys, return key, and tab key buttons on the PS3 controller?

**Comment by free5ty1e** [comment]
Yep!  Either via sixad connection profiles or using joy2key or something similar...
